src/app/pages/question/question.component.ts

```diff
@@ -94,18 +94,37 @@ export class QuestionComponent implements OnInit, CanDeactivate<CanComponentDeac
       this.currentIndex$,
     ]).pipe(
       map(
-        ([ questions, index ]) => questions?.[index] || {
-          question: '',
-          type: false,
-          difficulty: '',
-          category: '',
-          correct_answer: '',
-          incorrect_answers: [],
-        },
+        ([ questions, index ]) => {
+          const question = questions?.[index] || {
+            question: '',
+            type: false,
+            difficulty: '',
+            category: '',
+            correct_answer: '',
+            incorrect_answers: [],
+          };
+          return this.decodeQuestion(question);
+        }
       ),
     );
   }
 
+  private decodeText(html: string): string {
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(html, 'text/html');
+    return doc.documentElement.textContent || html;
+  }
```

Comment:
In such cases we tend to add a separate file as utiles or service to make this functionality `decodeText()` be available for the whole project. Let's follow such an approach, you can put it to shared